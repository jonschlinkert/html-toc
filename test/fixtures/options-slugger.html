<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <title>Document</title>
</head>

<body>
  <div id="toc"></div>
  <h1>abc ABC</h1>
</body>

</html>
