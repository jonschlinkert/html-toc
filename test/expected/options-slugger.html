<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <title>Document</title>
</head>

<body>
  <div id="toc"><ul class="nav sidenav"><li><a href="#abcABC">abc ABC</a></li></ul></div>
  <h1 id="abcABC">abc ABC<a href="#abcABC" name="abcABC" class="anchor">
  <span class="anchor-target" id="abcABC"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h1>
</body>

</html>
